satsense/util/calc.py

```diff
@@ -0,0 +1,11 @@
+import numpy as np
+from numba import jit
+
+
+@jit(nopython=True)
+def count_codewords(codewords, vector_length):
```

Comment:
This looks like [`numpy.bincount`](https://docs.scipy.org/doc/numpy/reference/generated/numpy.bincount.html), would it be possible to use that function instead?

Reply:
I didn't know of this functions existence... Yeah, it looks like an exact match so that should be preferable to use.